.github/workflows/main.yml

```diff
@@ -13,7 +13,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: [3.6, 3.7, 3.8, 3.9]
+        python: 
+          - "3.6"
+          - "3.7"
+          - "3.8"
+          - "3.9"
+          - "3.10"
```

Comment:
Could we use the reusable workflows here?

Reply:
good thinking, I added them to the worker charm, but didnt get around to this one:
https://github.com/charmed-kubernetes/charm-kubernetes-worker/pull/122/files

Reply:
as you wish